frontend/sozisel-app/src/components/TemplateCreation/EventsList/EventsListElement/EventListElement.tsx

```diff
@@ -0,0 +1,144 @@
+import "./EventListElement.scss";
+
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  IconButton,
+  Typography,
+} from "@material-ui/core";
+import React, { FC, ReactElement, useState } from "react";
+
+import DeleteIcon from "@material-ui/icons/Delete";
+import EditIcon from "@material-ui/icons/Edit";
+import { Event } from "../../../../model/Template";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import HelpOutlineIcon from "@material-ui/icons/HelpOutline";
+import Poll from "./Poll";
+import { Poll as PollData } from "../../../../graphql";
+import QuizQuestion from "./Quiz/Question";
+import { useEventContext } from "../../../../contexts/Event/EventContext";
+import { useTranslation } from "react-i18next";
+
+const EventHeaders: FC<{ event: Event }> = ({ event }) => {
+  const { t } = useTranslation("common");
+
+  switch (event.eventData.__typename) {
+    case "Quiz": {
+      return (
+        <>
+          <Typography>
+            {t(
+              "components.TemplateCreation.EventList.Quiz.percentageOfParticipants",
+              { value: event.eventData.targetPercentageOfParticipants }
+            )}
+          </Typography>
+        </>
+      );
+    }
+    case "Poll": {
+      return <></>;
```

Comment:
Is that intentional?

Reply:
Yes, Poll type does not generate any additional typography so in the preview card it is empty. Wanted to make the element optional for every event type as it has been already added for polls but the component was not generic (the typography was hard coded without a thought in mind that other modules would not have certain fields).